option.go

```diff
@@ -96,6 +97,15 @@ func WithTrustRemote() Option {
 	return &trustRemoteOption{}
 }
 
+// WithTraceMetadataAttributes enables metadata attributes for the metadata keys provided.
+// Attributes will be added as Trace attributes only.
+func WithTraceMetadataAttributes(requestKeys, responseKeys []string) Option {
+	return &traceMetadataAttributeOption{
+		metadataReqKeys: requestKeys,
+		metadataResKeys: responseKeys,
+	}
+}
```

Comment:
Unless we're really unsure whether we're talking about HTTP headers or trailers, I'd like to remove the word "Metadata" from connect-related libraries. Can we instead have `WithTraceRequestHeader(...string)` and `WithTraceResponseHeader(... string)` as separate options? This opens up space for us to later add options to capture response trailers too (which gRPC also calls "metadata").

Reply:
Fixed in cd1c0ee0e314c8ca16d25799fc4c96bd66cf4064